model/model.js

```diff
@@ -1,5 +1,23 @@
 const db = require("../db/connection");
 
+const QueryStringFront = `SELECT articles.article_id,
```

Comment:
We shouldn't have these variables declared in the outer scope of where they are being used, they are only relevant to the one endpoint so should be contained within that model.